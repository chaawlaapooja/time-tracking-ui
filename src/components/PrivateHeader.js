import React from 'react';

const PrivateHeader = (props) => {
  return (
    <div className="header">
        <div className="header__content">
            <h1 className="header__title">{props.title}</h1>
            <h2 style={{padding:'1rem'}}>Created by Pooja Chawla</h2>
        </div>
    </div>
  );
};

export default PrivateHeader;
