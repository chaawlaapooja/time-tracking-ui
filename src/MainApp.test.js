import React from 'react';
import PrivateHeader from './components/PrivateHeader';
import { cleanup, fireEvent, render } from '@testing-library/react';
import renderer from 'react-test-renderer';

// it('renders without crashing',()=>{
//     const div = document.createElement('div');
//     ReactDOM.render(<MainApp />, div)
// })
afterEach(cleanup)

it('app has private header',()=>{
    render(<PrivateHeader/>)
})

it('matches snapshot',()=>{
    const tree = renderer.create(<PrivateHeader title='Task time tracking app'/>).toJSON();
    expect(tree).toMatchSnapshot()
})