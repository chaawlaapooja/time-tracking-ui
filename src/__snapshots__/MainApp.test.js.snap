// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="header"
>
  <div
    className="header__content"
  >
    <h1
      className="header__title"
    >
      Task time tracking app
    </h1>
    <h2
      style={
        Object {
          "padding": "1rem",
        }
      }
    >
      Created by Pooja Chawla
    </h2>
  </div>
</div>
`;
